docs/installation.rst

```diff
@@ -18,7 +18,7 @@ Prerequisites
 .. _compatibility link : https://docs.nvidia.com/deploy/cuda-compatibility/index.html#enhanced-compat-minor-releases
 
 1. Linux x64.
-2. |driver link|_ supporting `CUDA 10.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e., 410.48 or later driver releases).
+2. |driver link|_ supporting `CUDA 11.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e., 450.80.02 or later driver releases).
 3. (Optional) One or more of the following deep learning frameworks:
 
   - |mxnet link|_ ``mxnet-cu100`` or later.
```

Comment:
```suggestion
  - |mxnet link|_ ``mxnet-cu110`` or later.
```
?

Reply:
All the framework version could be looked at. Either we should remove them or update them.

Reply:
Maybe we can remove the version as we test with a fixed version but NGC containers test with the latest. Also, this info doesn't tell anything meaningful to the users. Usually, you get the version you like most and try to use DALI with it, and so far rarely anyone complains.

Reply:
Yeah, I already removed the mention of a DLFW container version, I can remove those as well. No one in their right mind would be checking those versions, and we pretty much work with anything recent.

Reply:
Cannot agree more.